### PR TITLE
Compile mmc driver as module and load it after the FPGA, so that SD m…

### PR DESCRIPTION
**Pull request by lbara**
…emory card actually shows up when it is inserted before the boot

### CODE REVIEW

**Comment by anonymous-travis-swiftnav** [comment]
## v1.1.0-develop-2017081201-8-g41a60f5
+ [Artifacts (HITL Dashboard Page)](http://hitl-dashboard.swiftnav.com/files/swiftnav-artifacts-pull-requests/piksi_buildroot/v1.1.0-develop-2017081201-8-g41a60f5/)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.1.0-develop-2017081201-8-g41a60f5/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.1.0-develop-2017081201-8-g41a60f5

**Comment by gsmcmullin** [comment]
Is the module not automatically loaded by the S99coldplug script?

In DEV configuration this will fail to copy firmware/FPGA images from the SD card: https://github.com/swift-nav/piksi_buildroot/blob/master/board/piksiv3/rootfs-overlay/etc/init.d/S75dev_boot#L71
I don't know if this mechanism is still being used by anyone. It may be safe to remove.

**Comment by gsmcmullin** [comment]
Superseded by #323

File: board/piksiv3/rootfs-overlay/etc/init.d/S86_mmc [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+name="mmc_driver"
+
+start() {
+  modprobe mmc_core
```
Comment (silverjam) [inline board/piksiv3/rootfs-overlay/etc/init.d/S86_mmc]: These don't seem to be the correct module names-- in S75dev_boot we run `modprobe sdhci_of_arasan`, copy the .elf, .bit, and .ini then unmount and remove the sdhci_of_arason module.

Comment (silverjam) [reply board/piksiv3/rootfs-overlay/etc/init.d/S86_mmc]: Never mind, didn't notice this PR was closed...

**Pull request closed without merging**